e2e/test/scenarios/cross-version/source/02-datamodel.cy.spec.js

```diff
@@ -21,7 +21,13 @@ it("should configure data model settings", () => {
   );
 
   cy.get(".AdminList").findByText("Orders").click();
-  cy.findByDisplayValue("Product ID").parent().find(".Icon-gear").click();
+
+  cy.findByDisplayValue("Product ID")
```

Comment:
This is a disgusting change required since newer versions of metabase did some extra <div> wrapping or something. I verified that it works on older versions.